app/models/member.rb

```diff
@@ -22,9 +22,11 @@ def name=(full_name)
   end
 
   def liquid_data
+    full_name = name
     attributes.merge({
-      name: name,
-      welcome_name: name.blank? ? email : name
+      name: full_name,
```

Comment:
Yeah, this is just me cramming in fixes for Paul's bugs cause I wanted to get them out the door ASAP.